Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
here a working tree of your idea of use FloatBits(union):
https://github.com/latot/ppsspp/tree/pointer